scripts/screens/cat_screens.py

```diff
@@ -670,26 +670,28 @@ def determine_previous_and_next_cat(self):
 
         previous_cat = 0
         next_cat = 0
+        current_cat_found = 0
         if self.the_cat.dead and not is_instructor and self.the_cat.df == game.clan.instructor.df and \
                 not (self.the_cat.outside or self.the_cat.exiled):
             previous_cat = game.clan.instructor.ID
 
         if is_instructor:
-            next_cat = 1
+            current_cat_found = 1
```

Comment:
Out of curiosity, what was causing the next-cat issue? I thought it might have been the special starclan cat list, which is what is used for the page, while the next-cat function uses the general all_cat list.